tests/frontend/modules/test_identifytertiaryobjects.py

```diff
@@ -357,17 +400,17 @@ def test_do_not_shrink_identical():
         child_count = measurements.get_current_measurement(
             parent_name, child_count_feature
         )
-        for parent, expected_child_count in ((1, 1), (2, 0), (3, 1), (4, 0)):
+        for parent, expected_child_count in ((1, 1), (2, 1), (3, 1), (4, 1)):
             assert child_count[parent - 1] == expected_child_count
-        for child in (1, 3):
+        for child in (1, 2, 3, 4):
             assert parent_of[child - 1] == child
 
     for location_feature in (
         M_LOCATION_CENTER_X,
         M_LOCATION_CENTER_Y,
     ):
         values = measurements.get_current_measurement(TERTIARY, location_feature)
-        assert numpy.all(numpy.isnan(values) == [False, True, False])
+        assert numpy.all(numpy.isnan(values) == [False, False, False, False])
```

Comment:
This test (merged 6 years ago) checked for ghost objects, expected the object to disappear, and have nan parent object numbers